hnobi/MyDiary-React
# f#162183274 Signup functionality

#### What does this PR do?
- Adds signup page and its functionality
#### Description of Task to be completed?
- Adds signup component
- Adds signup reducer and actions
#### How should this be manually tested?
- git checkout staging
- git pull 
- git checkout ft-signup-162183274 
- Start the application using npm start
- Go to to [localhost:8080](http://localhost:8080)
-
- Steps anyone can take to check feature implementation
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/162183274
#### Screenshots (if appropriate)
#### Questions:`
